### PR TITLE
Add new page to Terminus manual

### LINKED ISSUE #2676: Document how to set default values for Terminus command options
Re: [Terminus Manual](https://pantheon.io/docs/terminus)
## Issue Description
It is possible to create configuration files to provide default values for the commandline options for any Terminus command; however, there is no documentation for this feature.

## Suggested Resolution
Add a "configuration" section to the Terminus manual.

See the related comment in the Terminus Pancakes plugin issue, https://github.com/terminus-plugin-project/terminus-pancakes-plugin/issues/19#issuecomment-316210837.

There is documentation on the configuration features used in Terminus in the [Configuration settings section of the Robo Getting Started guide](http://robo.li/getting-started/#configuration).

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #2676

## Effect
PR includes the following changes:
- New Terminus Manual page on configuration files.

## Remaining Work
- [ ] Finish Writing
- [x] Technical Review from @greg-1-anderson 
- [x] Copy Review

### CODE REVIEW

File: source/_docs/terminus/examples.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ terminustoc: true
 type: terminuspage
 layout: terminuspage
 nexturl: terminus/commands/
-previousurl: terminus/install/
+previousurl: terminus/configuratio/
```
Comment (greg-1-anderson) [inline source/_docs/terminus/examples.md]: * configuration

**Comment by alexfornuto** [comment]
@greg-1-anderson @rachelwhitton This is a very rough draft at this point. I'm hoping for some input on what other sections to add, how to expand, etc. Thoughts?

**Comment by greg-1-anderson** [comment]
+1 LGTM. It's a good MVP; not sure what else to add. Maybe 🚢  it and enhance as users ask more questions.

**Comment by alexfornuto** [comment]
I see that I've duplicated work already started by @obicke in #2893. Will consolidate tomorrow.

File: source/_docs/terminus/configuration.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    login:
+      options:
+        email: anita@example.org
+  site:
+    pancakes:
+      options:
+        app: sequelpro
+```
+
+The example above does two things:
```
Comment (greg-1-anderson) [inline source/_docs/terminus/configuration.md]: Config option hierarchy is explained here: https://github.com/consolidation/config#providing-command-options

Feel free to copy in any of that text, or just link there

Comment (alexfornuto) [reply source/_docs/terminus/configuration.md]: Are we using that repositories code/tools in Terminus to make this work?

Comment (greg-1-anderson) [reply source/_docs/terminus/configuration.md]: Yes. Terminus is built on top of Robo. Robo uses consolidation/config for its config system. Terminus' config inherits from Robo's config.

**Pull request merged**